DNN Platform/Modules/DnnExportImport/Components/Services/PagesExportService.cs

```diff
@@ -979,6 +979,18 @@ private int ImportTabModulesAndRelatedItems(TabInfo localTab, ExportTab otherTab
 
             return count;
         }
+        /*
+            Update Modules.IsDeleted with ExportModule.IsDeleted and not ExportTabModule.IsDeleted. 
+            ExportTabModule.IsDeleted may different from ExportModule.IsDeleted when Module is deleted.
+            Change ModuleInfo.IsDeleted to ExportModule.IsDeleted and reverting to ExportMabModule.IsDeleted after 
+            updating Modules.
+        */
+        private void UpdateModuleWithIsDeletedHandling(ExportTabModule other, ExportModule otherModule, ModuleInfo local)
```

Comment:
I would like to get better names for these parameters as it is a bit hard to follow what is being done.

Possible suggestion:
other = exportedTabModule
otherModule = exportedModule
local = importedModule

Reply:
Okay. these looks a lot better.